test/swar/BasicOperations.cpp

```diff
@@ -81,66 +162,62 @@ HE(2, u8, 0xAA, 0x2);
+  SWAR<8, u32> Mplier{0xA050301};
+  auto Expected = 0x320F0601;
+  auto result = multiplication_OverflowUnsafe_SpecificBitCount_deprecated<4>(
+      Micand, Mplier);
+  CHECK(Expected == result.value());
 }
 
 TEST_CASE("Parity", "[swar]") {
-    // For each nibble, E indicates (E)ven and O (O)dd parities
-    //                EEOEEOOO
-    auto Examples = 0xFF13A7E4;
-    SWAR<4, u32> casesBy4{Examples};
-    SWAR<8, u32> casesBy8{Examples};
-    auto by4 = parity(casesBy4);
-    auto by8 = parity(casesBy8);
-    CHECK(by4.value() == 0x00800888);
-    CHECK(by8.value() == 0x00808000);
+  // For each nibble, E indicates (E)ven and O (O)dd parities
+  //                EEOEEOOO
+  auto Examples = 0xFF13A7E4;
+  SWAR<4, u32> casesBy4{Examples};
+  SWAR<8, u32> casesBy8{Examples};
+  auto by4 = parity(casesBy4);
+  auto by8 = parity(casesBy8);
+  CHECK(by4.value() == 0x00800888);
+  CHECK(by8.value() == 0x00808000);
 }
 
-TEST_CASE(
-    "Isolate",
-    "[swar]"
-) {
-    for (auto i = 0; i < 63; ++i) {
-        CHECK(i == isolate<8>(i));
-        CHECK(i == isolate<8>(0xFF00+i));
-        CHECK(i == isolate<8>(0xFFFF00+i));
-    }
-    for (auto i = 0; i < 31; ++i) {
-        CHECK(i == isolate<7>(i));
-        CHECK(i == isolate<7>(0xFF00+i));
-        CHECK(i == isolate<7>(0xFFFF00+i));
-    }
-    for (auto i = 0; i < 31; ++i) {
-        CHECK(i == isolate<11>(i));
-        CHECK(i == isolate<11>(0xF800+i));
-        CHECK(i == isolate<11>(0xFFF800+i));
-    }
+TEST_CASE("Isolate", "[swar]") {
+  for (auto i = 0; i < 63; ++i) {
+    CHECK(i == isolate<8>(i));
+    CHECK(i == isolate<8>(0xFF00 + i));
+    CHECK(i == isolate<8>(0xFFFF00 + i));
+  }
+  for (auto i = 0; i < 31; ++i) {
+    CHECK(i == isolate<7>(i));
+    CHECK(i == isolate<7>(0xFF00 + i));
+    CHECK(i == isolate<7>(0xFFFF00 + i));
+  }
+  for (auto i = 0; i < 31; ++i) {
+    CHECK(i == isolate<11>(i));
+    CHECK(i == isolate<11>(0xF800 + i));
+    CHECK(i == isolate<11>(0xFFF800 + i));
+  }
 }
 
 TEST_CASE("Compress/Expand", "[swar]") {
-    unsigned
-        Mask =   0b0001'0011'0111'0111'0110'1110'1100'1010,
-        ToMove = 0b0101'0101'0101'0101'0101'0101'0101'0101,
-        // Selection: 1   01  101  101  10  010  01   0 0
-        result = 0b0001'0'1'1'0'1'1'0'1'10'0'10'0'1'0'0;
-    auto q = compress(S32_32{ToMove}, S32_32{Mask});
-    CHECK(result == q.value());
-    SECTION("Regression 1") {
-        u64
-            input =   0b1010'1001'0110'0001'1001'0000'0010'1010'0100'0111'1110'1001'1111'0001'1110'1011,
-            mask =    0b0110'0000'0001'0101'0101'1111'0101'1100'0110'1111'0100'0111'0001'1000'0101'0010,
-            expected =0b0001'0000'0000'0001'0001'0000'0000'0010'0010'0111'0001'0001'0001'0000'0010'0001;
-        using S = S4_64;
-        auto v = compress(S{input}, S{mask});
-        CHECK(expected == v.value());
-    }
+  unsigned Mask = 0b0001'0011'0111'0111'0110'1110'1100'1010,
+           ToMove = 0b0101'0101'0101'0101'0101'0101'0101'0101,
```

Comment:
No!!!
That accidental alignment style is not acceptable!!

Reply:
gah sorry this is my auto-formatter. i think it would be valuable to have a clang-format to avoid style issues like this in the future. it at least is valuable to have this to minimise the frequency of merge conflicts.